README.md

```diff
@@ -19,10 +19,74 @@ Tools used to generate the website:
    Asciidoc documents from different sources in the [Camel](https://github.com/apache/camel) 
    and [Camel K](https://github.com/apache/camel-k) repositories where user manual and component 
    reference documentation resides and renders them for inclusion in this website.
+ - [Maven](https://maven.apache.org/) (optional) a build tool used to run the complete website generating process
 
-## Building the website
+## Build with Maven
```

Comment:
If building with Maven is optional it should be the last mentioned method here. First method should be the using Node and Yarn.

Reply:
Agreed! Let me change the ordering